src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/ApplicationsTransformer.kt

```diff
@@ -175,13 +174,6 @@ class ApplicationsTransformer(
     }
   }
 
-  private fun getStatusFromSummary(entity: DomainCas2ApplicationSummary): ApplicationStatus {
```

Comment:
Yes you're right.  I've added some more info to the commit message.

Since the Cas2ApplicationSummary interface is no longer required it has been removed, along with a legacy function getStatusFromSummary() in ApplicationsTransformer which isn't used (even prior to this work).